src/mesh/PacketHistory.h

```diff
@@ -16,8 +16,9 @@ class PacketHistory
         PacketId id;
         uint32_t rxTimeMsec;              // Unix time in msecs - the time we received it,  0 means empty
         uint8_t next_hop;                 // The next hop asked for this packet
+        uint8_t hop_limit;                // The hop limit when we received this packet
         uint8_t relayed_by[NUM_RELAYERS]; // Array of nodes that relayed this packet
-    };                                    // 4B + 4B + 4B + 1B + 3B = 16B
+    };                                    // 4B + 4B + 4B + 1B + 1B + 3B = 17B (will be padded to 20B)
```

Comment:
Interesting, I was working on an optimization of the NextHopRouter and also needed this entry (although then the hop limit with which we sent it out). Since it will be padded to 20B, we can increase `NUM_RELAYERS` to 6 for free.